test/lib/trace.spec.js

```diff
@@ -2345,6 +2345,139 @@ describe("lib/trace", () => {
         });
       });
     });
+
+    describe("modern ESM imports", () => {
```

Comment:
I typically do the nitty-gritty feature testing on `traceFile` and leave `traceFiles` tests to things involving multiple `srcPaths`, so I moved the existing tests in this describe block and targeted `traceFile`.

The tests provided great coverage, and I just added a few more corner cases to keep everything covered 😄

Reply:
thanks!